README.md

```diff
@@ -401,7 +401,42 @@ public class Example {
 }
 ```
 
-### `JSONObject countEntity(JSONObject data)`
+### `JSONObject countEntityTotal(Collection<String> tables)`
+Count the total number of inserted entities in the given tables. This method corresponds to a [POST request at /query/count/entity/total](http://panel.slicingdice.com/docs/#api-details-api-endpoints-get-query-count-entity-total).
+
+#### Request example
+
+```java
+import com.slicingdice.jslicer.SlicingDice;
+import java.io.IOException;
+import java.util.ArrayList;
+import org.json.JSONObject;
+
+public class Example {
+
+    public static void main(String[] args) throws IOException {
+        SlicingDice slicingDice = new SlicingDice("MASTER_OR_READ_API_KEY", false);
+        ArrayList<String> tables = new ArrayList<>();
```

Comment:
Change it to https://docs.oracle.com/javase/7/docs/api/java/util/Collections.html#singleton(T)